### PR TITLE
Minor fixes to make it work

### PR DESCRIPTION
**Pull request by clangenb**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +1,7 @@
 [submodule "shared/BEST-Energy"]
 	path = shared/BEST-Energy
 	url = git@github.com:olisystems/BEST-Energy.git
+	branch = cl/oli-pycli
```
Comment (clangenb) [inline .gitmodules:4]: This is a simple hotfix branch, where I made some stuff public.

File: Cargo.lock
Comment (clangenb) [inline Cargo.lock:1]: If you had committed it, you could have saved me some trouble. :)

It is in general best practice to commit the lock file for binaries to have a deterministic build.

Comment (j-ti) [reply Cargo.lock:1]: ok, sorry for that

Comment (clangenb) [reply Cargo.lock:1]: No, worries, all good. At least I could uncover an issue in our codebase because of that. :)

File: .gitmodules (hunk 1 above)
Comment (j-ti) [reply .gitmodules:4]: Is it recommended to update main with this fixing branch or rather keep this branch updated with ongoing changes? Both client-dependency-fixing-branch and the current worker main branch should be able to work together despite changes regarding the fixes right?

**Pull request merged**